images/miq-app/docker-assets/appliance-initialize.sh

```diff
@@ -11,6 +11,9 @@ sleep "${APPLICATION_INIT_DELAY}"
 # Prepare initialization environment
 prepare_init_env
 
+# Generate the certs if needed
+${APPLIANCE_ROOT}/initialize_appliance.sh
+
```

Comment:
I think I'm just going to add the `--key` back to the `appliance_console_cli` command and remove it from this script.

I'll just have to also add it to https://github.com/ManageIQ/manageiq-appliance-build/blob/master/kickstarts/partials/post/db_init.ks.erb#L7